source/features/keyboard-shortcuts.js

```diff
@@ -0,0 +1,50 @@
+export default async () => {
+	const customShortcuts = [
+		{
+			name: 'Actions',
+			description: 'Shortcuts for common actions.',
+			shortcuts: [
+				{
+					keys: [
+						'Ctrl',
+						'm'
+					],
+					description: 'Toggle Night Mode'
+				}
+			]
+		},
+		{
+			name: 'Navigation',
+			description: 'Shortcuts for navigating between items in timelines.',
+			shortcuts: []
+		},
+		{
+			name: 'Timelines',
+			description: 'Shortcuts for navigating to different timelines or pages.',
+			shortcuts: []
+		}
+	];
+
+	document.addEventListener('keydown', event => {
+		const keyName = event.key;
+		switch (keyName) {
+			case event.ctrlKey && 'm':
+				if (document.querySelector('.nightmode-toggle')) {
+					document.querySelector('.nightmode-toggle').click();
+				}
```

Comment:
Can you move this to a separate function that is called here? It's not large right now, but it will become unmaintainable like this when we add more shortcuts.

Reply:
Maybe make `keyboard-shortcuts` a folder. This file would become the index and any shortcut added would have it's function on it's own file. Something like the tree below.

```
source
└── features
   └── keyboard-shortcuts
      ├── index.js
      └── nightmode-toggle.js
```

Reply:
Just pushed a version with the night mode toggle in a separate function, but the 'keyboard-shortcuts folder' option does seem like a better solution.